Core/src/main/java/com/plotsquared/core/permissions/Permission.java

```diff
@@ -37,7 +37,9 @@
     PERMISSION_STAR("*"),
     PERMISSION_ADMIN("plots.admin"),
     PERMISSION_ADMIN_AREA_SUDO("plots.admin.area.sudo"),
+    @Deprecated(forRemoval = true)
     PERMISSION_PROJECTILE_UNOWNED("plots.projectile.unowned"),
+    @Deprecated(forRemoval = true)
```

Comment:
```suggestion
    @Deprecated(forRemoval = true, since = "6.3.0")
    PERMISSION_PROJECTILE_UNOWNED("plots.projectile.unowned"),
    @Deprecated(forRemoval = true, since = "6.3.0")
```